### PR TITLE
BZ #1198973: Add MTU to all interfaces involved in tenant network

### PR DESCRIPTION
**Pull request by bcrochet**
If an interface is involved in the tenant network, then it needs to have
the MTU stated on it. This includes physical interfaces and bonds.

### CODE REVIEW

File: app/lib/staypuft/network_query.rb [hunk 1]
```diff
@@ -59,6 +59,23 @@ def subnet_for_host(subnet_type_name, host=@host)
       interface_hash_for_host(subnet_type_name, host)[:subnet]
     end
 
+    def interfaces_for_tenant_subnet(host=@host)
+      subnet = subnet_for_host(Staypuft::SubnetType::TENANT, host)
+      top_level_interface = host.interfaces.where(identifier:
+        interface_hash_for_host(Staypuft::SubnetType::TENANT,
+          host)[:interface]).first
```
Comment (sseago) [inline app/lib/staypuft/network_query.rb]: subnet_for_host calls interface_hash_for_host internally, so you're making this twice. Perhaps you should do this instead:
interface_hash = interface_hash_for_host(Staypuft::SubnetType::TENANT, host)
subnet = interface_hash[:subnet]
top_level_interface = host.interfaces.where(identifier: interface_hash[:interface]).first

Comment (sseago) [inline app/lib/staypuft/network_query.rb]: Also, make sure this works in cases where there is no tenant interface on the host (i.e. subnet and interface are nil) --

**Comment by sseago** [comment]
Looks good now with the changes.

**Comment by sseago** [comment]
One more visual ACK...

**Comment by knowncitizen** [comment]
NICE, ack.

**Pull request merged**